<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cclips</class>
 <widget class="QDialog" name="cclips">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>507</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	
	background-color: rgb(255, 255, 255);
}

QFrame#frame{
	border-image: url(:/res/goodBKG2.jpg);
}

#frame_2{
	border-radius: 30px;
	background-color: rgba(158, 158, 158, 120);
}

#frame_3{
	border-radius: 30px;
	background-color: rgba(158, 158, 158, 120);
}

#frame_4{
	border-radius: 30px;
	background-color: rgba(158, 158, 158, 120);
}

QLabel{
	background: transparent;
	color: rgba(255, 255, 255, 200);
	font: 12pt &quot;等线&quot;;
}

QPushButton{
	border-radius: 10px;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(100, 197, 255, 255), stop:1 rgba(133, 71, 255, 255));
	
	font: 12pt &quot;等线&quot;;
}

QPushButton:hover{
	border-radius: 10px;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 197, 255, 255), stop:1 rgba(133, 71, 255, 255));
	
	font: 12pt &quot;等线&quot;;
}

QPushButton:pressed{
	border-radius: 10px;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(100, 197, 255, 255), stop:1 rgba(133, 71, 255, 255));
	
	font: 12pt &quot;等线&quot;;
}

QMessageBox {
	background-color: #F2F2F2;
}

QMessageBox QLabel#qt_msgbox_label { 
	color: #298DFF;
	background-color: transparent;
	min-width: 240px; 
	min-height: 40px; 
}

QMessageBox QLabel#qt_msgboxex_icon_label { 
	width: 40px;
	height: 40px; 
}

QMessageBox QPushButton { 
	border: 1px solid #298DFF;
	border-radius: 3px;
	background-color: #F2F2F2;
	color: #298DFF;
	font-family: &quot;Microsoft YaHei&quot;;
	font-size: 10pt;
	min-width: 70px;
	min-height: 25px;
}

QMessageBox QPushButton:hover {
	background-color: #298DFF;
	color: #F2F2F2;
}

QMessageBox QPushButton:pressed {
	background-color: #257FE6;
}

QMessageBox QDialogButtonBox#qt_msgbox_buttonbox {
	button-layout: 0; 
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>511</width>
        <height>371</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <widget class="MySlider" name="positionSlider">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>90</y>
         <width>401</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QSlider::groove:horizontal { 
border: 1px solid #bbb; 
background: white; 
height: 10px; 
border-radius: 4px; 
} 
   
QSlider::sub-page:horizontal { 
background: qlineargradient(x1: 0, y1: 0.2, x2: 1, y2: 1, 
    stop: 0 #bbf, stop: 1 #55f); 
border: 1px solid #777; 
height: 10px; 
border-radius: 4px; 
} 
   
QSlider::add-page:horizontal { 
background: #fff; 
border: 1px solid #777; 
height: 10px; 
border-radius: 4px; 
} 
   
QSlider::handle:horizontal { 
background: qlineargradient(x1:0, y1:0, x2:1, y2:1, 
    stop:0 #eee, stop:1 #ccc); 
border: 1px solid #777; 
width: 13px; 
margin-top: -2px; 
margin-bottom: -2px; 
border-radius: 4px; 
} 
   
QSlider::handle:horizontal:hover { 
background: qlineargradient(x1:0, y1:0, x2:1, y2:1, 
    stop:0 #fff, stop:1 #ddd); 
border: 1px solid #444; 
border-radius: 4px; 
} 
   
QSlider::sub-page:horizontal:disabled { 
background: #bbb; 
border-color: #999; 
} 
   
QSlider::add-page:horizontal:disabled { 
background: #eee; 
border-color: #999; 
} 
   
QSlider::handle:horizontal:disabled { 
background: #eee; 
border: 1px solid #aaa; 
border-radius: 4px; 
} </string>
       </property>
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
      </widget>
      <widget class="QPushButton" name="fadeInButton">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>290</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>淡入</string>
       </property>
      </widget>
      <widget class="QPushButton" name="fadeOutButton">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>290</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>淡出</string>
       </property>
      </widget>
      <widget class="QPushButton" name="playButton">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>120</y>
         <width>41</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="QIcon::ThemeIcon::MediaPlaybackStart"/>
       </property>
      </widget>
      <widget class="QPushButton" name="pauseButton">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>120</y>
         <width>41</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="QIcon::ThemeIcon::MediaPlaybackPause"/>
       </property>
      </widget>
      <widget class="QLabel" name="speedLabel">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>220</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>速度: 1.00x</string>
       </property>
      </widget>
      <widget class="QLabel" name="volumeLabel">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>220</y>
         <width>161</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>音量：50%</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>播放区</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>190</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>控制区</string>
       </property>
      </widget>
      <widget class="MySlider" name="speedSlider">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>250</y>
         <width>161</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QSlider::groove:horizontal { 
border: 1px solid #bbb; 
background: white; 
height: 10px; 
border-radius: 4px; 
} 
   
QSlider::sub-page:horizontal { 
background: qlineargradient(x1: 0, y1: 0.2, x2: 1, y2: 1, 
    stop: 0 #bbf, stop: 1 #55f); 
border: 1px solid #777; 
height: 10px; 
border-radius: 4px; 
} 
   
QSlider::add-page:horizontal { 
background: #fff; 
border: 1px solid #777; 
height: 10px; 
border-radius: 4px; 
} 
   
QSlider::handle:horizontal { 
background: qlineargradient(x1:0, y1:0, x2:1, y2:1, 
    stop:0 #eee, stop:1 #ccc); 
border: 1px solid #777; 
width: 13px; 
margin-top: -2px; 
margin-bottom: -2px; 
border-radius: 4px; 
} 
   
QSlider::handle:horizontal:hover { 
background: qlineargradient(x1:0, y1:0, x2:1, y2:1, 
    stop:0 #fff, stop:1 #ddd); 
border: 1px solid #444; 
border-radius: 4px; 
} 
   
QSlider::sub-page:horizontal:disabled { 
background: #bbb; 
border-color: #999; 
} 
   
QSlider::add-page:horizontal:disabled { 
background: #eee; 
border-color: #999; 
} 
   
QSlider::handle:horizontal:disabled { 
background: #eee; 
border: 1px solid #aaa; 
border-radius: 4px; 
} </string>
       </property>
       <property name="minimum">
        <number>50</number>
       </property>
       <property name="maximum">
        <number>200</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
      </widget>
      <widget class="MySlider" name="volumeSlider">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>250</y>
         <width>161</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QSlider::groove:horizontal { 
border: 1px solid #bbb; 
background: white; 
height: 10px; 
border-radius: 4px; 
} 
   
QSlider::sub-page:horizontal { 
background: qlineargradient(x1: 0, y1: 0.2, x2: 1, y2: 1, 
    stop: 0 #bbf, stop: 1 #55f); 
border: 1px solid #777; 
height: 10px; 
border-radius: 4px; 
} 
   
QSlider::add-page:horizontal { 
background: #fff; 
border: 1px solid #777; 
height: 10px; 
border-radius: 4px; 
} 
   
QSlider::handle:horizontal { 
background: qlineargradient(x1:0, y1:0, x2:1, y2:1, 
    stop:0 #eee, stop:1 #ccc); 
border: 1px solid #777; 
width: 13px; 
margin-top: -2px; 
margin-bottom: -2px; 
border-radius: 4px; 
} 
   
QSlider::handle:horizontal:hover { 
background: qlineargradient(x1:0, y1:0, x2:1, y2:1, 
    stop:0 #fff, stop:1 #ddd); 
border: 1px solid #444; 
border-radius: 4px; 
} 
   
QSlider::sub-page:horizontal:disabled { 
background: #bbb; 
border-color: #999; 
} 
   
QSlider::add-page:horizontal:disabled { 
background: #eee; 
border-color: #999; 
} 
   
QSlider::handle:horizontal:disabled { 
background: #eee; 
border: 1px solid #aaa; 
border-radius: 4px; 
} </string>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>50</number>
       </property>
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="position">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>60</y>
         <width>36</width>
         <height>28</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>36</width>
         <height>28</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>36</width>
         <height>28</height>
        </size>
       </property>
       <property name="text">
        <string>00:00</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>60</y>
         <width>4</width>
         <height>28</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>4</width>
         <height>28</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>4</width>
         <height>28</height>
        </size>
       </property>
       <property name="text">
        <string>/</string>
       </property>
      </widget>
      <widget class="QLabel" name="duration">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>60</y>
         <width>36</width>
         <height>28</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>36</width>
         <height>28</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>36</width>
         <height>28</height>
        </size>
       </property>
       <property name="text">
        <string>00:00</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>330</y>
         <width>211</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Tips：淡出后需重新调整音量</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="cin">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>20</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>151</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>151</width>
        <height>31</height>
       </size>
      </property>
      <property name="text">
       <string>导入音乐</string>
      </property>
     </widget>
     <widget class="QPushButton" name="saveButton">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>440</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>151</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>151</width>
        <height>31</height>
       </size>
      </property>
      <property name="text">
       <string>保存</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>470</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Tips：保存的是原文件哦</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MySlider</class>
   <extends>QSlider</extends>
   <header>myslider.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
